Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN        bash /tmp/env-config.sh
 # cache is empty, so we retry once if it failed
 COPY       ./package.json ./npm-shrinkwrap.json /calypso/
 RUN        true \
-           && npm install --production || npm install --production \
+           && npm install --production \
```

Comment:
so trusting. 🥇

Reply:
They've been selling us deterministic installs, right? 😅

Reply:
😆

Reply:
what does the diff here mean?

Reply:
I removed the comment describing it as well:

```sh
# Sometimes "npm install" fails the first time when the		
# cache is empty, so we retry once if it failed
```

The first `npm install` was failing on occasion, this just retries in case it doesn't complete cleanly. The retry was removed.

Reply:
thanks for the explanation! (and removing the comment)